andela/thorin-cfh
# #156882168 User should be able to view a game log of past games

#### What does this PR do?

Lets user view a game log of past games

#### Description of Task to be completed?
 - Add API endpoint to get games history
- Add API endpoint to get leaderboard
- Add API endpoint to get donations
- Design user interface to view the game log

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm start` and open the application on the browser. Click on the dashboard link on the index page and you should be redirected to the dashboard where you will see the gamelog, donations and leaderboard.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#156882168

#### Screenshots (if appropriate)

#### Questions: N/A
